lib/build.js

```diff
@@ -214,6 +214,14 @@ function createModel(pathResolver, modelConfig, options, config, jsBundleOptions
                 cssBundleOptions
             )
         )
+        .then(() => !options.allowDownload && utils.section('Disabling report download option in bundles', () => {
+            fs.writeFileSync(
+                pathResolver('model.js'),
+                fs.readFileSync(pathResolver('model.js'), 'utf8')
+                    .replace(new RegExp(`function ${utils.MODE_PLACEHOLDER}\\(\\){return.*?}\\(\\)`), '"single"')
```

Comment:
Not sure we need change this

Reply:
We're doing this for each model separately. So if we disable download for the static-builded report we need to disable the ability to switch models as well.

Reply:
Yeah, I've got it, you were right) fixed it